generators/app/templates/_package.json

```diff
@@ -10,8 +10,8 @@
     "serve:sw": "<%= props.packageManager %> run build<%= props.packageManager === 'npm' ? ' -s' : '' %> && npx http-server ./<%= props.target.includes('cordova') ? 'www': 'dist' %> -p 4200",
 <% } -%>
     "lint": "ng lint --type-check && stylelint \"src/**/*.scss\" --syntax scss && htmlhint \"src\" --config .htmlhintrc",
-    "test": "ng test",
-    "test:ci": "<%= props.packageManager %> run lint<%= props.packageManager === 'npm' ? ' -s' : '' %> && ng test --single-run --code-coverage",
+    "test": "ngx-scripts env npm_package_version && ng test",
```

Comment:
could you use `<%= props.packageManager %> run env<%= props.packageManager === 'npm' ? ' -s' : '' %>` like start/build scripts? so that it's possible to add env variable by modifying only the `env` scripts

Reply:
Sorry If I forgot to write it, but could you do the same for the `test` script? Then it will be good to merge 😉